src/Client.php

```diff
@@ -3,63 +3,270 @@
 namespace Dvsa\Authentication\Ldap;
 
 use Dvsa\Contracts\Auth\AccessTokenInterface;
+use Dvsa\Contracts\Auth\Exceptions\ClientException;
 use Dvsa\Contracts\Auth\OAuthClientInterface;
 use Dvsa\Contracts\Auth\ResourceOwnerInterface;
+use Illuminate\Support\Str;
+use Symfony\Component\Ldap\Entry;
+use Symfony\Component\Ldap\Exception\ExceptionInterface;
+use Symfony\Component\Ldap\LdapInterface;
 
 class Client implements OAuthClientInterface
 {
+    /**
+     * @var int
+     */
+    public static $tokenExpiry = 86400;
```

Comment:
Is this and the one below used at all? Or am i just being blind?

Reply:
No no, my bad, good spot, I'll get it removed.